source/dub/dependency.d

```diff
@@ -397,6 +425,7 @@ struct Dependency {
 		A specification is valid if it can match at least one version.
 	*/
 	bool valid() const {
+		if (isGit) return true;
```

Comment:
I suggest being consistent in usage: `this.isGit` (as elsewhere), not just `isGit`